moveit_experimental/CMakeLists.txt

```diff
@@ -24,6 +24,11 @@ COMPONENTS
   std_msgs
   trajectory_msgs
   visualization_msgs
+  # The following are needed by jog_arm:
+  moveit_msgs
```

Comment:
duplicate, remove

please alphabetize as the others already are

remove comment